frontend/src/citizen-frontend/messages/utils.ts

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+export const sessionKeepalive = async () => {
+  const response = await fetch('/api/application/auth/status', {
```

Comment:
this could maybe be moved to lib-commons with some parameterisation if the only difference is the api base path, but not necessary by any means

Reply:
Tarvitaanko uutta fetch-pohjaista kutsua tälle? Meillähän on jo auth status endpointille funktio